app/services/notifications/also_commented_service.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Notifications
+  class AlsoCommentedService
+    def self.notify(comment, _)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for notify is too high. [<4, 20, 2> 20.49/20]